src/app/board/board.component.ts

```diff
@@ -6,8 +6,8 @@ import { WorkItemService } from '../work-item/work-item.service';
 
 @Component({
   selector: 'my-board',
-  templateUrl: '/board.component.html',
-  styleUrls: ['/board.component.scss']
+  templateUrl: './board.component.html',
+  styleUrls: ['./board.component.scss']
```

Comment:
They do not appear to fail but you are right, the "./" is the standard way and so I updated it to match that standard.